CombineDemo/CombineDemoTests/Publishers/BuiltInPublishers/JustTests.swift

```diff
@@ -0,0 +1,104 @@
+//
+//  JustTests.swift
+//  CombineDemoTests
+//
+//  Created by Manish Rathi on 27/03/2023.
+//
+
+import Foundation
+import Combine
+import XCTest
+
+final class JustTests: XCTestCase {
+    var justPublisher: Just<String>!
+    var subject: JustTestingViewModel<String>!
+
+    override func setUp() {
+        super.setUp()
+
+        justPublisher = Just("James bond")
+        subject = JustTestingViewModel(justPublisher)
+    }
+
+    override func tearDown() {
+        subject.cancellables.forEach { $0.cancel() }
+        subject.cancellables.removeAll()
```

Comment:
This is no need to do.
The concept of `cancellables` is about if it deallocated it will cancel all subscriptions which it contains

Reply:
Good to know! thank you